### DISCUSSION TITLE
I want to show lint as failure but still allow the pr to be merged is there a way to do that?

### DISCUSSION THREAD

**Discussion by ssambhav108** [super-linter/super-linter#5531] [Q&A]
I want to show lint as failure but still allow the pr to be merged is there a way to do that?
So the way is that it shows red on lint but still allows a merge on that?

**Comment by ferrarimarco** [comment]
Hi @ssambhav108 ! You can achieve that by not setting super-linter as a required check when configuring branch protection rules for your repository. It's a GitHub feature, and not specific to super-linter.
